docs/source/tutorial_notebooks/DREGS_tutorial_NERSC.ipynb

```diff
@@ -0,0 +1,371 @@
+    "\n",
+    "Note that in the original dataset we did not specify `name` during registration. The default name for a dataset is the file or directory name taken from its relative path. In our example above this would be `my_desc_dataset`.\n",
+    "\n",
+    "The combination of `name`, `version` and `version_suffix` for any dataset in DREGS must be unique. As we are updating a dataset with the same name, we have to make sure to update the version to a new value. One handy feature is automatic version \"bumping\" for datasets, i.e., rather than specifying a new version string manually, one can select \"major\", \"minor\" or \"patch\" for the version string to automatically bump it up. In our case, selecting \"minor\" will automatically generate the version \"1.1.0\"."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 6,
+   "id": "88a01c97-a9f5-49b5-b8de-83712ac5f7f0",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Add new entry for an updated dataset with an updated version.\n",
+    "dataset_id = dregs.Registrar.register_dataset(\n",
+    "    \"dregs_nersc_tutorial/my_updated_desc_dataset\",\n",
+    "    \"minor\", # Automatically bumps to \"1.1.0\"\n",
+    "    description=\"An output from some DESC code (updated)\",\n",
+    "    is_overwritable=True,\n",
+    "    old_location=\"dummy_dataset.txt\",\n",
+    "    name=\"my_desc_dataset\" # Using this name links it to the previous dataset.\n",
+    ")"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "3bf6d38e-ea5e-4916-aa5d-654b2587942d",
+   "metadata": {},
+   "source": [
+    "## Querying the data registry\n",
+    "\n",
+    "Now that we have covered the basics of dataset registration, we can have a look at how to query entries in the DREGS database.\n",
+    "\n",
+    "Queries are constructed from one or more boolean logic \"filters\", which translate to SQL `WHERE` clauses in the code. \n",
+    "\n",
+    "For example, to create a filter that will query for all datasets in DREGS with the name \"my_desc_dataset\" would be as follows:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": 7,
+   "id": "f9901d89-b1d7-48c9-8110-ce16ecba3a7e",
+   "metadata": {
+    "tags": []
+   },
+   "outputs": [],
+   "source": [
+    "# Create a filter that queries on the dataset name\n",
+    "f = dregs.Query.gen_filter('dataset.name', '==', 'my_desc_dataset')"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "305a8df6-6967-4280-a5e8-6ea8831eff09",
+   "metadata": {},
+   "source": [
+    "Like with SQL, we can either be explicit, or not, with our table names. For example we could have filtered on `name` rather than explicitly saying we want to query on `name` from the `dataset` table (i.e., `dataset.name`). However we would always recommend being explicit by including the table name with filters.\n",
```

Comment:
I think if just `name` were specified an exception would be raised since more than one table has a column names `name`. We need to make clear that table can be omitted _only_ if the column name is unique.  And therefore we recommend, to be on the safe side, that column name always be qualified by table.

Reply:
tried to clarify